File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        int i, j;
        if(a >= b){
            j = a;
            i = b;
        }else{
            i = a;
            j = b;
        }
        for(int k = i; k <= j; k++){
            answer += k;
        }
        return answer;
    }
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.kt
class Solution {
    fun solution(a: Int, b: Int): Long {
        var answer: Long = 0
        val start = minOf(a, b)
        val end = maxOf(a, b)
        
        for (n in start..end) {
            answer += n
        }
        
        return answer
    }
}
